stable/node-problem-detector/templates/psp-clusterrolebinding.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.rbac.pspEnabled }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ template "node-problem-detector.fullname" . }}-psp
+  labels:
+    app: {{ include "node-problem-detector.name" . }}
```

Comment:
These are the old fashioned labels. And in `stable/node-problem-detector/templates/psp.yaml` you've used the new style. I think make them all new style for consistency.